Libplanet/State/IAccountStateDelta.cs

```diff
@@ -136,8 +139,15 @@ public interface IAccountStateDelta : IAccountStateView
         /// <exception cref="InsufficientBalanceException">Thrown when the <paramref name="sender"/>
         /// has insufficient balance than <paramref name="value"/> to transfer and
         /// the <paramref name="allowNegativeBalance"/> option is turned off.</exception>
+        /// <remarks>
+        /// The behavior is different depending on <paramref name="context"/>'s
+        /// <see cref="IActionContext.BlockProtocolVersion"/>.  There is a bug for version 0
+        /// where this may not act as intended.  Such behavior is left intact for backward
+        /// compatibility.
+        /// </remarks>
         [Pure]
         IAccountStateDelta TransferAsset(
+            IActionContext context,
```

Comment:
If `IActionContext` need on `TransaferAsset` for ACL, How about `MintAsset` or `BurnAsset`? They are needless `IActionContext`?

Reply:
That will be dealt with in a separate PR, which is almost ready. 🙃
Coming right up after this gets merged. 😗

Reply:
Also, although both `TransferAsset` and the `MintAsset` and `BurnAsset` pair depend on `IActionContext`, the way these depend on `IActionContext` are different. As for the former, the dependency is baked in to the implementation `type`, and for the latter, the dependency is baked in to a hidden variable named `Signer`. 😑